vmdb/app/models/mixins/ems_openstack_mixin.rb

```diff
@@ -24,7 +24,7 @@ def browser_url
   def openstack_handle(options = {})
     require 'openstack/openstack_handle'
     @openstack_handle ||= begin
-      raise "no credentials defined" if self.missing_credentials?(options[:auth_type])
+      raise MiqException::MiqInvalidCredentialsError, "No credentials defined" if self.missing_credentials?(options[:auth_type])
```

Comment:
@jrafanie What did we decide on this from your other thread?  I think we wanted to introduce a new Exception class, because technically they are different.